tests/jballerina-unit-test/src/test/resources/test-src/object/object-field-defaultable.bal

```diff
@@ -16,3 +31,37 @@ class Person {
         self.name = name;
     }
 }
+
+class ErrorField {
+    public error 'error;
+    public int 'int;
+
+    function init (error er, int value = 20) {
+        self.'error = er;
+        self.'int = value;
+    }
+}
+
+public function testErrorAsObjectField () {
```

Comment:
Is this the best file to add this test?

```suggestion
public function testErrorAsObjectField() {
```

Reply:
Suggestions?

Reply:
Maybe under `org.ballerinalang.test.types` or quoted identifier tests?